lib/creek/styles/converter.rb

```diff
@@ -45,16 +46,18 @@ def self.call(value, type, style, options = {})
         when 'b'
           value.to_i == 1
         when 'str'
-          value
+          convert_html_entities(value)
```

Comment:
it looks like this is somewhat correct, the ST_Xstring type (http://www.datypic.com/sc/ooxml/t-ssml_ST_Xstring.html) inherits from xsd:string (http://www.datypic.com/sc/xsd/t-xsd_string.html)
which requires entities for < and & at least.

I'm not sure this should be decoding all "html" entities though, because this is xml not html. From what I understand, XML only supports 5 entities by default: quot, amp, apos, lt, gt.